docker_run.sh

```diff
@@ -0,0 +1,16 @@
+
+WORKSPACE=`pwd`
+WORK_DIR='/workdir'
+
+docker run -v $WORKSPACE:/workdir \
```

Comment:
Running like this won't be handling stopping by a signal.

i.e. if those run would be stopped by CI those docker instances would keep running
See dtest run_test.sh or dbuild to see how to support signal trapping.

Reply:
@fruch can you point to what is missing? detach=true?
I would like to merge and give it a try, if needed to send a fix on top of it.

Reply:
yes detach=true, and the rest of the code, calling `docker log -f {container_id}` and handling signals, so it can be stopped without leaving the docker running

https://github.com/scylladb/scylla-dtest/blob/next/scripts/run_test.sh#L200